diff --git a/docs/Mobile SDK Installation/nativescript/sensitive-data-occlusion-nativescript.md b/docs/Mobile SDK Installation/nativescript/sensitive-data-occlusion-nativescript.md
--- a/docs/Mobile SDK Installation/nativescript/sensitive-data-occlusion-nativescript.md
+++ b/docs/Mobile SDK Installation/nativescript/sensitive-data-occlusion-nativescript.md
@@ -280,9 +280,9 @@
 <br />
 
 ### Sensitive Data Occlusion Inside WebViews
 
 When dealing with sensitive data in WebViews, it's essential to ensure that any personal or sensitive information is properly occluded to maintain user privacy. Our guide provides step-by-step instructions on how to handle occlusion within WebViews effectively.
 
 For more information, click the button below:
 
-[Sensitive Data Occlusion in WebViews Documentation](/docs/ALL UXCAM APIs/screen-blurring/sensitive-views-inside-webviews)+[Sensitive Data Occlusion in WebViews Documentation](<../../ALL UXCAM APIs/screen-blurring/sensitive-views-inside-webviews.md>)
